extra/2-piping.js

```diff
@@ -16,26 +16,27 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  return num1 + num2;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  return a * b;
 }
 
-function format() {
-
+function format(amount) {
+  return `£${parseFloat(amount.toFixed(1))}`;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = format(multiply(add(startingValue, 10), 2));
```

Comment:
Can you articulate why this code would be bad practice?

Reply:
My take on this one was that chaining too many times would be considered as bad practice.